### PR TITLE
[stable/traefik] Add ability to enable hostPort setup

### PR DESCRIPTION
**Pull request by aslafy-z**
This will allow us to configure traefik to use hostPort.

### CODE REVIEW

**Comment by aslafy-z** [comment]
CLA done

**Comment by mattfarina** [comment]
@aslafy-z Can you try commenting again to catch the CLA. If that doesn't work, please make sure the CLA site is linked to your github account.

**Comment by aslafy-z** [comment]
I need you, CLA ❤️

**Comment by aslafy-z** [comment]
CLA done

**Comment by krancour** [comment]
~~This seems pretty reasonable. LGTM.~~ Version bump is wrong.

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.14.5
+version: 1.14.6
```
Comment (krancour) [inline stable/traefik/Chart.yaml]: Since this adds a feature, we should be bumping to 1.15.0.

**Pull request merged**